.eslintrc.yml

```diff
@@ -121,6 +122,7 @@ rules:
   no-process-exit: off
   node/no-unsupported-features/es-syntax: off
   node/shebang: off
+  node/no-missing-import: off # Doesn't like .js imports of .interfaces.ts files
```

Comment:
Same reason as above. Besides, with ESM, all modules are imported before the code even runs, meaning you will get any missing module errors in the console and the app crashing